### PR TITLE
Merging development branch into main, Implementing the identity service

### PR DESCRIPTION
**Pull request by SaintAngeLs**
Adding the Identity service

### CODE REVIEW

**Comment by SaintAngeLs** [comment]
@eggwhat, @an2508374, @SaintAngeLs, I would like with a kindly mind and opened soul invite you to review the code.

File: SwifttParcel.Services.Identity/scripts/start.sh
Comment (SaintAngeLs) [inline SwifttParcel.Services.Identity/scripts/start.sh:1]: Adding the scripts to run (the chmod +x _name_of_script_.sh may be important!)

**Review by SaintAngeLs — COMMENTED** [review]
Adding the Identity Microservice

**Review by SaintAngeLs — COMMENTED** [review]
Adding necessary assemblies

File: SwifttParcel.Services.Identity/Dockerfile
Comment (eggwhat) [inline SwifttParcel.Services.Identity/Dockerfile:1]: Did you try to docker build? Line nr 9 should probably be commented.

File: .gitignore
Comment (eggwhat) [inline .gitignore:1]: Can we use this instead https://github.com/github/gitignore/blob/main/VisualStudio.gitignore

**Review by an2508374 — APPROVED** [review]

File: SwiftParcel.Web/backend/.gitignore
Comment (SaintAngeLs) [inline SwiftParcel.Web/backend/.gitignore:1]: Updating the gitignore

File: .gitignore
Comment (SaintAngeLs) [reply .gitignore:1]: Yeap, already updated it in the development branch

File: SwifttParcel.Services.Identity/Dockerfile
Comment (SaintAngeLs) [reply SwifttParcel.Services.Identity/Dockerfile:1]: You are totally correct, the dockerfile here is only like the dummy one, to make it use the in the future when at leas 2 service will we written and the docker-compose.yml file with all the services like the vault, rebitMQ, ... will also be added in the next commit. Now I do use the containers of  all the services only locally

Comment (SaintAngeLs) [inline SwifttParcel.Services.Identity/Dockerfile:1]: The **line 9** should definitely immediately be _desperately_ updated (@eggwhat thank you)

**Review by SaintAngeLs — COMMENTED** [review]
Updating the Dockefile

**Comment by eggwhat** [comment]
I am pretty sure that because of the folder structure (e.g. _SwiftParcel.Services.Identity.Api/SwiftParcel.Services.Identity.Api/_, same thing for other folders) .gitignore is not working properly. I suggest that we delete one layer of directory depth, or just change the names.

**Comment by SaintAngeLs** [comment]
> I am pretty sure that because of the folder structure (e.g. _SwiftParcel.Services.Identity.Api/SwiftParcel.Services.Identity.Api/_, same thing for other folders) .gitignore is not working properly. I suggest that we delete one layer of directory depth, or just change the names.

I agree with the request to update the path to the specific projects. I will be updated soo

**Comment by SaintAngeLs** [comment]
> ### This branch has no conflicts with the base branch
> Merging can be performed automatically.

The possible one more reason might be the fact, that the .gitignore was added with the hardcoded way, with no specific way defined in the .net project

**Comment by SaintAngeLs** [comment]
@eggwhat, @an2508374 .git ignore was updated(obj and the bin directories are removed from the git gistory). The structure of the project is not changed still.

**Comment by SaintAngeLs** [comment]
The language statistics will be shown only in the **main** branch after merging

**Review by an2508374 — APPROVED** [review]

**Comment by eggwhat** [comment]
> @eggwhat, @an2508374 .git ignore was updated(obj and the bin directories are removed from the git gistory). The structure of the project is not changed still.

It seems like it works properly now, thank you.

**Review by eggwhat — APPROVED** [review]
There are some remnants of the outdated code but I guess it's a good place to get going. Good job!

**Pull request merged**